src/plone/restapi/tests/test_atcontent_serializer.py

```diff
@@ -110,3 +110,32 @@ def test_serializer_includes_folder_items(self):
             'description': '',
             'title': u'A Document'},
             obj['items'][1])
+
+    def test_serializer_orders_folder_items_by_gopip(self):
```

Comment:
@lukasgraf Just a minor issue. I would prefer to use "test_serializer_orders_folder_items_by_get_object_position_in_parent". A reader will not get what "gopip" means.